_sass/_logo.scss

```diff
@@ -0,0 +1,221 @@
+article.post.logo{
+
+#stretch{
+	@include span-columns(3);
+	@include omega(2n);
+	height: 20em;
+}
+#squash{
+	@include span-columns(9);
+	height: 6em;
+}
+
+#logo-white-on-black{
+	@include span-columns(6);
+	rect{fill:$bvn-black;}
+	path{fill:$bvn-white;}
+}
+#logo-black-on-white{
+	@include span-columns(6);
+	@include omega();
+	rect{fill:$bvn-white;}
+	path{fill:$bvn-black;}
+}
+
+#stripped-background{
+	@include span-columns(6);
+	.background{		fill: #1E8260;		}
+	.foreground{		fill: #F0D100;		}
+	.bvn-text-colour{	fill: #FFFFFF;		}
+}
+
+#pink-background{
+	@include span-columns(6);
+	@include omega();
+	.background{		fill: #E9ABC8;			}
+	.foreground{		fill: #E9ABC8;			}
+	.bvn-text-colour{	fill: #BEE2E9;			}
+}	
+#logo-size-width{
+	@include span-columns(6);
+	@include omega();
+}
+
+#logo-graphic-elements{
+	@include span-columns(6);
+	@include omega();
+}
+
+#logo-clear-space{
+	.logo-one-third-clear-space{	visibility:visible;		}
+	.logo-surrounding-rectangle{	visibility:visible;		}
+	.logo-top-circle{				visibility:visible;		}
+	.logo-guide-lines{				visibility:hidden;		}
+	.logo-bvn-text{					visibility:visible;		}
+}
+
+#logo-logo-with-address-one{
+	@include span-columns(6);
+	
+}
+
+#logo-logo-with-address-two{
+	@include span-columns(6);
+	@include omega();
+}
+
+#logo-logo-with-address-three{
+	@include span-columns(6);
+}
+
+#logo-geometrical-form{
+  .logo-structure-surrounding-box{ opacity:1;}
+  .logo-structure-black-box{ opacity:0;}
+  .logo-structure-white-box{ opacity:0;}
+  .logo-structure-bvn-text{ opacity:.3;}
+  .logo-structure-frame{ opacity:0;}
+  .logo-structure-circle-top{ opacity:1;}
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Colon after property should be followed by one space<br>Opening curly brace `{` should be preceded by one space